### DISCUSSION TITLE
Options To Not Make Ride Entrance and Exit Invisble When Using See Through Rides

### DISCUSSION THREAD

**Discussion by jcurbaniak** [OpenRCT2/OpenRCT2#22474] [Feature requests]
When using See Through Ride option I would like to have a way to NOT make the ride Entrance and/or Exit invisible.   In a congested situation, it can be very difficult to build paths that connect properly with a downloaded ride.  If the view of these is obstructed by the ride, the only option I have is to move either or both Entrance and Exit somewhere else.   Or not use the ride at all.

**Comment by J-PIE-314** [comment]
You could use the horizontal clipping option in the cut-away view options.

![screenshot of cut-away view](https://github.com/user-attachments/assets/6c4afa87-47a8-4bd8-b3af-f64b4d0d633a)

**Comment by jcurbaniak** [comment]
This seems very helpful.   Do you have any detail on how to select the area?

 

Jerry U.

**Reply by J-PIE-314** [reply]
1. From the visibility settings drop-down menu, choose "Cut-away view".
2. Click on the button "Select area" (as shown in the screenshot below).

![clipping 2](https://github.com/user-attachments/assets/3c3b0dfa-5e52-44c3-92c4-1f891072e380)

3. In the viewport, click and drag the only area you want to see.

To disable the effect, either click the button "Clear selection" or close the cut-away view window.

**Comment by jcurbaniak** [comment]
Thanks.  I understood the first 2 steps.

 

On #3 is that a horizontal drag or something else/

 

Jerry Urbaniak

**Comment by jcurbaniak** [comment]
OK.  I think I figured it out.   The white matrix boxes during the drag could not easily be seen in a congested area.

 

Thanks for all of your help.

 

Jerry U.

 

From: ***@***.*** ***@***.***> 
Sent: Thursday, August 8, 2024 10:08 AM
To: 'OpenRCT2/OpenRCT2' ***@***.***>
Subject: RE: [OpenRCT2/OpenRCT2] Options To Not Make Ride Entrance and Exit Invisble When Using See Through Rides (Discussion #22474)

 

Thanks.  I understood the first 2 steps.

 

On #3 is that a horizontal drag or something else/

 

Jerry Urbaniak
